src/main/scala/epic/logo/FeatureVector.scala

```diff
@@ -0,0 +1,50 @@
+package epic.logo
+
+import scala.collection.JavaConversions._
+import breeze.linalg._
+import breeze.math.MutableInnerProductModule
+
+case class FeatureVector[W](var data :W)(implicit space: MutableInnerProductModule[W, Double]) {
```

Comment:
it's not clear this class actually needs to exist, to me, but again, letting this through